packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -50,6 +50,18 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
         @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService
     ) { }
 
+    registerTermCommands(termWidget: TerminalWidget, commands: CommandRegistry) {
+        const CLEAR_OUTPUT: Command = {
```

Comment:
please mover it to `TerminalCommands` namespace

Reply:
Moved